### DISCUSSION TITLE
Retrieving "latest" driver data

### DISCUSSION THREAD

**Discussion by connor-rose** [br-g/openf1#21] [Q&A]
I'm trying to retrieve the driver data for latest meetings however I noticed when I retrieve data from https://api.openf1.org/v1/drivers?meeting_key=latest many of the data fields, such as first name, last name, team colors, are null. Is that design, if so should I be using a different url to get up to date driver data?

**Comment by br-g** [comment]
Hello,
Thanks for reporting this issue!
I will have a closer look at this soon

**Comment by br-g** [comment]
Fixed!
